src/Backend/Modules/Pages/Resources/views/FormLayout.html.twig

```diff
@@ -139,52 +135,80 @@
+        {% do form.title.setRendered() %}
+        {% do form.titleOverwrite.setRendered() %}
       </div>
+
       <div class="checkboxTextFieldCombo">
-        <div class="form-group">
-          {{ form_widget(form.parent.children.navigation.children.titleOverwrite) }}
-        </div>
-        <div class="form-group">
-          {{ form_label(form.parent.children.navigation.children.title) }}
-          {{ form_widget(form.parent.children.navigation.children.title) }}
-          {{ form_errors(form.parent.children.navigation.children.title) }}
-          <small class="form-text text-muted">{{ 'msg.HelpNavigationTitle'|trans|raw }}</small>
-        </div>
+        <checkbox-enable-field
+          switch-check
+          ref="{{ form.parent.navigation.title.vars.id }}"
+          checkbox-id="{{ form.parent.navigation.titleOverwrite.vars.id }}"
+          checkbox-label="{{ form.parent.navigation.titleOverwrite.vars.label|trans|ucfirst }}"
+          checkbox-name="{{ form.parent.navigation.titleOverwrite.vars.full_name }}"
+          field-id="{{ form.parent.navigation.title.vars.id }}"
+          field-label="{{ form.parent.navigation.title.vars.label|trans|ucfirst }}"
+          field-name="{{ form.parent.navigation.title.vars.full_name }}"
+          field-help-text="{{ 'msg.HelpNavigationTitle'|trans|raw }}"
+          value-on-load="{{ form.parent.vars.value.navigationTitle }}"
+          {% if form.parent.navigation.titleOverwrite.vars.checked %} enabled-on-load {% endif %}
+        >
+        </checkbox-enable-field>
+        {% do form.parent.navigation.title.setRendered() %}
+        {% do form.parent.navigation.titleOverwrite.setRendered() %}
       </div>
     </fieldset>
 
     <fieldset>
       <legend>{{ 'lbl.MetaInformation'|trans|ucfirst }}</legend>
       <div class="checkboxTextFieldCombo">
-        <div class="form-group">
-          {{ form_widget(form.descriptionOverwrite) }}
-        </div>
-        <div class="form-group">
-          {{ form_label(form.description) }}
-          {{ form_widget(form.description) }}
-          {{ form_errors(form.description) }}
-          <small class="form-text text-muted">{{ 'msg.HelpMetaDescription'|trans|raw }}</small>
-        </div>
+        <checkbox-enable-field
+          switch-check
+          ref="{{ form.description.vars.id }}"
+          checkbox-id="{{ form.descriptionOverwrite.vars.id }}"
+          checkbox-label="{{ form.descriptionOverwrite.vars.label|trans|capitalize }}"
+          checkbox-name="{{ form.descriptionOverwrite.vars.full_name }}"
+          field-id="{{ form.description.vars.id }}"
+          field-label="{{  form.description.vars.label|trans|capitalize }}"
+          field-name="{{ form.description.vars.full_name }}"
+          field-help-text="{{  'msg.HelpMetaDescription'|trans|raw }}"
+          value-on-load="{{ form.description.vars.value }}"
+          {% if form.descriptionOverwrite.vars.checked %} enabled-on-load {% endif %}
```

Comment:
whitespace after `enabled-on-load ` to much, this issue appears multiply time in this file and the other files where the same code was added

Reply:
fixed ae55c25edd5b331110f8a54c002734192f656ca1